### ISSUE TITLE
Enhance uploaded file status text read by screenreaders

### ISSUE THREAD

**Issue by ggeisler** [pod4lib/aggregator#854]
Related to PR #853. In that PR I added "Type of file: " to the existing `visually-hidden` uploaded file status text:

```
<span class="visually-hidden">Type of file: <%= file.pod_metadata_status %></span>
```

That appears in the first column of the Upload activity (Activity page) and Uploaded files (Provider home) tables.

<img width="1137" alt="Screen Shot 2022-06-01 at 5 06 36 PM" src="https://user-images.githubusercontent.com/101482/171521183-e08136b7-b863-4dfd-8f18-b93436270606.png">

---

However, the `file.pod_metadata_status` value is not informative enough for a screenreader user. The current values are one-word values like "unknown" and "success". To help the screenreader user who doesn't have visual access to the file status icon legend above the table, we should use the legend values for `file.pod_metadata_status`.

In other words, for a file that currently has a rendered value of `success` it should instead be `Valid MARC (adds, updates, or deletes)` so the screenreader user hears "Type of file: Valid MARC (adds, updates, or deletes)".
